client/src/components/Menu/Menu.js

```diff
@@ -1,8 +1,19 @@
 import React, { Component } from "react";
 import { Link } from "react-router-dom";
+import { getSessionUser, getUserRoles } from "../../helpers/api.js";
 
 class Menu extends Component {
+  state = { admin: false };
+  isThisAdmin = async ()=>{
+    const userData = await getSessionUser();
+    const { data: roles } = await getUserRoles(userData.user_id);
+    const userRoles = roles.map(role => role.role);
+    if (userRoles.includes("Admin")) {
+      this.setState({ admin: true });    }
+   }
+
   render() {
+    this.isThisAdmin();
```

Comment:
And if you use componentDidMount, then you don't need to call this function here.